components/google_sheets/sources/new-row-added/new-row-added.js

```diff
@@ -0,0 +1,172 @@
+const google_sheets = require("../../google_sheets.app.js");
+const google_drive = require("../../google_drive.app.js");
```

Comment:
`const google_drive = require("../../../google_drive/google_drive.app.js");` should work here